systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -169,7 +169,7 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
                         .endGenericKafkaListener()
                     .endListeners()
                     .withNewInlineLogging()
-                        .addToLoggers("log4j.rootLogger", "DEBUG")
+                        .addToLoggers("log4j.rootLogger", "DEBUG, CONSOLE")
```

Comment:
Aren't we removing this in the other PRs? Please check with @stanlyDoge and @mstruk. It is IIRC also not what the docs suggest ... we should primarily follow our docs to make sure the docs are valid and test them this way as well.

Reply:
Leftover, when I was debugging some issues.

Reply:
Like @scholzj mentioned, I also guess that this is part of another PR, I don't know if @Frawless didn't do something similar.

Reply:
PR fixing this is not merged yet https://github.com/strimzi/strimzi-kafka-operator/pull/3672